FileIO/GMSInterface.cpp

```diff
@@ -317,7 +321,17 @@ MeshLib::Mesh* GMSInterface::readGMS3DMMesh(const std::string &filename)
 	INFO("GMSInterface::readGMS3DMMesh(): \tfinished.");
 
 	const std::string mesh_name = BaseLib::extractBaseNameWithoutExtension(filename);
-	return new MeshLib::Mesh(mesh_name, nodes, elements);
+	MeshLib::Properties properties;
+	if (mat_ids.size() == elements.size()) {
+		boost::optional<MeshLib::PropertyVector<int> &> opt_pv
```

Comment:
A message that material ids are going to be ignored (in the other case) could be helpful.